docroot/themes/humsci/su_humsci_gin_admin/su_humsci_gin_admin.info.yml

```diff
@@ -0,0 +1,27 @@
+name: SU HumSci Gin Admin
+type: theme
+base theme: gin
+description: "Gin specific overrides unique to Stanford Humsci"
+core_version_requirement: ^8.8 || ^9
+version: '1.0'
+hidden: false
+
+libraries:
+  - claro/global-styling
+  - gin/gin
+  - su_humsci_gin_admin/global-styling
+  - su_humsci_gin_admin/fix-autocomplete-width
+
+# Regions
+regions:
+  header: "Header"
+  pre_content: "Pre-content"
+  breadcrumb: Breadcrumb
+  highlighted: Highlighted
+  help: Help
+  content: Content
+  page_top: "Page top"
+  page_bottom: "Page bottom"
+  sidebar_first: "First sidebar"
+regions_hidden:
```

Comment:
Why hide it? it's just an admin theme.

Reply:
This is the default setting in the gin theme: https://git.drupalcode.org/project/gin/-/blob/8.x-3.x/gin.info.yml

Reply:
interesting. I wonder why bother. :shrug: